_posts/_posts/0000-01-02-RickGiannini.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![I-AM-NOT-PREPARED-FOR-THIS](https://user-images.githubusercontent.com/79111260/107996607-c4b32700-6fae-11eb-8b51-1ac5aef9d342.png)

Reply:
This is going to be a rough semester